### PR TITLE
[stable/sonarqube] add annotations

### PR DESCRIPTION
**Pull request by AWKIF**
## What this PR does / why we need it:

Add annotations to pvc and pods
ie: allow to add annotations to pvc to keep ressource in case of helm delete

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Review by mathieupotier — APPROVED** [review]

**Comment by AWKIF** [comment]
/retest
ui-test timeout before app started

File: stable/sonarqube/templates/pvc.yaml [hunk 1]
```diff
@@ -8,6 +8,12 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+{{ if .Values.presistence.annotations}}
```
Comment (happiness9721) [inline stable/sonarqube/templates/pvc.yaml:11]: typo here😂

Comment (AWKIF) [reply stable/sonarqube/templates/pvc.yaml:11]: haaa :(

Comment (AWKIF) [reply stable/sonarqube/templates/pvc.yaml:11]: #15807 
fix here

Comment (hectoralicea) [reply stable/sonarqube/templates/pvc.yaml:11]: had this error as well.  I thought this was the stable branch.

**Pull request merged**